src/components/GameItem.js

```diff
@@ -16,10 +24,38 @@ class GameItem extends Component {
     };
 
     // Update this to select the correct icon for each item
-    const icon = ItemIcons.rock;
+    let icon;
+    if (this.props.itemType === "rock") {
+      icon = ItemIcons.rock;
+    }
+
+    if (this.props.itemType === "litter") {
+      icon = ItemIcons.litter;
+    }
+
+    if (this.props.itemType === "bush") {
+      icon = ItemIcons.bush;
+    }
+
+    if (this.props.itemType === "flower") {
+      icon = ItemIcons.flower;
+    }
+
+    if (this.props.itemType === "mushroom") {
+      icon = ItemIcons.mushroom;
+    }
```

Comment:
Instead of a series of `if` statements, maybe `if ... else if` statements, or the JavaScript switch statement would be better here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch